src/cow-react/modules/swap/pure/CurrencySelectButton/index.tsx

```diff
@@ -24,7 +24,7 @@ export function CurrencySelectButton(props: CurrencySelectButtonProps) {
       <styledEl.CurrencySymbol className="token-symbol-container" stubbed={stubbed}>
         {currency ? currency.symbol : <Trans>Select a token</Trans>}
       </styledEl.CurrencySymbol>
-      {stubbed ? <styledEl.ArrowDownStubbed /> : <styledEl.ArrowDown />}
+      {stubbed ? <styledEl.ArrowDown stubbed /> : <styledEl.ArrowDown />}
```

Comment:
From the slack thread I understood you would rename this. Is that coming in another PR?